pom.xml

```diff
@@ -11,8 +11,7 @@
   <version>1.0.0</version>
 
   <!--
-    CAUTION: Temporal applications require Java 8 (1.8) compiler source
-             and target at the time this material was developed.
+    CAUTION: Temporal applications require Java 8 (1.8) or higher.
```

Comment:
I feel like this warning may be best if it is put in the README as a prerequisite instead? Though I don't know the habits of Java developers or if they are likely to be checking the pom.xml for warnings like this. Feel free to disregard if it feels redundant.